src/test/fedora-32

```diff
@@ -0,0 +1 @@
+fedora-31
```

Comment:
could you explain in the commit message what the support is for ?

also i'd suggest squash these "test: Add support for *" commits into a single one.